### DISCUSSION TITLE
Can't reach the WebUI

### DISCUSSION THREAD

**Discussion by Thomasje96** [haugene/docker-transmission-openvpn#2854] [Q&A]
Hello everyone,

I am very, very new to the community and the journey has been very fun and challenging so far.

I have finally managed to get everything up and running. VPN works like a charm. i had some issue with mullvad (ipv6 issue, apparently) so i am now using PIA. I am, however, now officially at a dead end and i have searched far and wide without any solution, or i am simply to stupid/inexperienced to circumvent/fix it. the debug page referred me here so here's to good luck.

So, as stated, everything is finally working like a charm. I am running it through the container manager app on my synology ds218+. i have confirmed that the VPN is working through a curl test. Now the problem: i can't access the port on [myipadress]:9091. Curl gives no response as well:

>curl: (7) Failed to connect to [myipadress] port 9091 after 2065 ms: Couldn't connect to server.

in the logs of the container however i can see that there is a valid response on [myipadress]:9091. 

I have set the LOCAL_NETWORK user to 192.168.0.0/24, as was given in the guide. i have also tried 192.168.1.0/16. I have no clue what's going wrong or why i can't access the UI, so any help is appreciated. Also, for some reason i can't extract logs but i'll try to give as much information as i can.

**Comment by pkishino** [comment]
LOCAL_NETWORK is depending on YOUR setup and your network. Please read the documentation on how to set this value correctly. 
is your ip range 192.168.0.0/24 ? If not then it won’t work for you and you need to set it to the correct value

**Reply by Thomasje96** [reply]
Hi pkishino,

Thank you for taking your time to answer the question. I am very dumb when it comes to this so i follow guides to a T. I have looked at guides to figure out my IP range but i don't understand the explanations. I know i have a default gateway of 192.168.1.1 and ipv4 gateway of  192.168.1.99, thats why i figured it would be one of the ranges i started in my original adress because 'same number good unga bunga'.

If you could help me figure out the range i would me more than grateful

Also as addage: couldnt find any info on how to do this in the documentation. The page on networking (where I'm constantly getting redirected too), doesn't really explain it and states 'To do', because it was moved

**Reply by pkishino** [reply]
Check here
https://duckduckgo.com/?q=what+subnet+am+i+on&t=iphone&ia=web
and then also check for your docker network as this can vary depending on platform etc.
You can add multiple signets to LOCAL_NETWORK if you are unsure
